### PR TITLE
Add title to checkbox to ensure service owners know they should only enable backup instance types feature with capacity-eng approval

### PR DESCRIPTION
**Pull request by liyaqin1**

### CODE REVIEW

**Review by osoriano — COMMENTED** [review]
Seems we could show red since users should not usually enable this. I see an example here:

Screenshot 2024-09-17 at 3 10 56 PM

**Review by osoriano — DISMISSED** [review]

**Comment by liyaqin1** [comment]
@osoriano I used warning instead of danger. What do you think? ^^
Screenshot 2024-09-17 at 5 47 46 PM

File: deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js [hunk 1]
```diff
@@ -585,12 +585,19 @@ Vue.component('backup-hosttypes', {
...
+        <form-warning :alert-text="backupTypesWarning"></form-warning>\
     </div>\
     </div>',
     props: ['backuphosttypes'],
     computed: {
         isVisible: function () {
             return this.backuphosttypes !== 'None'
+        },
+        backupTypesWarning: function () {
+            const part1 = "You should only enable this feature with capacity-eng approval.";
```
Comment (osoriano) [inline deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js:597]:
```suggestion
            const part1 = "You should only enable this feature with capacity-eng approval. ";
```

**Review by osoriano — DISMISSED** [review]

File: deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js [hunk 2]
```diff
@@ -585,12 +585,19 @@ Vue.component('backup-hosttypes', {
...
     </div>',
     props: ['backuphosttypes'],
     computed: {
         isVisible: function () {
             return this.backuphosttypes !== 'None'
+        },
+        backupTypesWarning: function () {
+            const part1 = "You should only enable this feature with capacity-eng approval. ";
+            const part2 = "Your enable/disable operation on this feature triggers a notification to capacity-eng.";
+            text = part1 + part2;
```
Comment (osoriano) [inline deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js:599]: Shouldn't this be `const text`?
```suggestion
            const text = part1 + part2;
```

**Review by osoriano — APPROVED** [review]

**Pull request merged**